packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -88,10 +106,25 @@ const ActionHeader = ({
   onPrevious,
   onNext,
   children,
+  backButtonA11yLabel,
+  nextButtonA11yLabel,
+  prevButtonA11yLabel,
   ...customProps
 }) => {
   const theme = React.useContext(ThemeContext);
 
+  const closeButtonId = `terra-action-header-close-button-${uniqueid()}`;
+  const maximizeButtonId = `terra-action-header-maximize-button-${uniqueid()}`;
+  const minimizeButtonId = `terra-action-header-minimize-button-${uniqueid()}`;
+
+  const visuallyHiddenComponent = (
+    <>
+      <VisuallyHiddenText aria-hidden id={closeButtonId} text={intl.formatMessage({ id: 'Terra.actionHeader.close.description' }, { text })} />
```

Comment:
Is `aria-hidden` added for any specific screen reader and browser combination..??

Reply:
Without aria-hidden screen reader will read these text twice